types.ts

```diff
@@ -502,12 +502,12 @@ export interface CustomAuthParameter {
  * user or system authentication). When constructing a network request, you may indicate where these values should
  * be inserted by our fetcher service using the syntax described below (similar to templating engines).
  *
- * {% raw %}
- * To insert the credentials, simply put `{{<paramName>-<invocationToken>}}` as a string anywhere in your request,
+ * \{% raw %\}
+ * To insert the credentials, simply put `\{\{<paramName>-<invocationToken>\}\}` as a string anywhere in your request,
```

Comment:
Is this intended to format different visually? If so (it does look different in the demo) maybe the example should have double-quotes for clarity so the reader can see that the back slashes are for string escaping?

Reply:
Good catch. I had to escape the braces for raw and endraw, but I shouldn't have escaped them in the inline code block. Undone.